liota/dccs/iotcc.py

```diff
@@ -422,6 +432,7 @@ def store_device_info(self, uuid, name, dev_type, prop_dict):
 
     def write_entity_file(self, prop_dict, res_uuid):
         file_path = self.entity_file_path + '/' + res_uuid + '.json'
+        prop_dict.update({"Entity_Timestamp": datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")})
```

Comment:
Yes, the reason is that I need to get os restart time, but this is just for internal usage. The file with Entity_Timestamp is internal file, which will not be exposed to AW agents. The files AW agents need is under another folder, which will be deleted once AW agents get them.

uptime library is os independent, and do not have gcc dependency, so it is a very good choice. But its API only returns datetime. I need to compare current time with this datetime. So should not use getUTCmillis().